starter-code/routes/drones.js

```diff
@@ -1,20 +1,27 @@
-const express = require('express');
+const express = require("express");
 
 // require the Drone model here
 
 const router = express.Router();
 
-
-router.get('/drones', (req, res, next) => {
+router.get("/drones", (req, res, next) => {
```

Comment:
Not:
router.get("/drones", (req, res, next) => {

use:
router.get("/", (req, res, next) => {